packages/expo/CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
 
 ### 🎉 New features
 
+- Added utterance word tracking support for iOS and Android. This allows the ability to highlight each word in an utterance. ([#20726](https://github.com/expo/expo/pull/20726) by [@gabrieljoelc](https://github.com/gabrieljoelc))
+
```

Comment:
```suggestion
```
Remove this. Bot suggested this changelog, because you edited files inside `packages/expo/`. Revert these version bumps and the bot will be ok with the changelog entry only in `packages/expo-speech/CHANGELOG.md`

Reply:
oh i see. my bad. will do.